runelite-client/src/main/java/net/runelite/client/game/FishingSpot.java

```diff
@@ -145,21 +197,108 @@
 		SPOTS = builder.build();
 	}
 
-	FishingSpot(String spot, int fishSpriteId, int... ids)
-	{
-		this(spot, spot, fishSpriteId, ids);
-	}
+	private final String name;
+	private final String worldMapTooltip;
+	private final ImmutableMap<FishingTools, Catches> catchMap;
+
+	private final int[] ids;
 
-	FishingSpot(String spot, String worldMapTooltip, int fishSpriteId, int... ids)
+	FishingSpot(String style, String worldMapTooltip, ImmutableMap<FishingTools, Catches> catchMap, int... ids)
 	{
-		this.name = spot;
+		this.name = style;
```

Comment:
https://oldschool.runescape.wiki/w/Fishing_spots
Bit unsure how I wanted to further label the fishing spots for better readability based off the entries above - I could change the enum names to reflect the styles rather than distinctive fish as it currently is - or I could just remove the style name all together, I just want to make it reasonably understandable

Reply:
I'd recommend changing the FishingSpot enum names instead